### DISCUSSION TITLE
[@types/react] Support for rendering `undefined` in React components

### DISCUSSION THREAD

**Discussion by PanzerKunst** [DefinitelyTyped/DefinitelyTyped#63777] [Issues with a @types package] [answered]
React 18 now [allows to render `undefined` in components](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#react).

It seems to me that in the latest version of @types/react, the TS interface `FunctionComponent` doesn't yet support it:

![image](https://user-images.githubusercontent.com/1530772/210246288-c220c424-e399-4ae0-934b-7684a93da261.png)

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
Duplicate of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051 and https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18912.

This has been a long standing issues for which [a solution was proposed](https://github.com/microsoft/TypeScript/pull/51328) but is missing a response from the TypeScript team.
